CHANGELOG.md

```diff
@@ -46,7 +46,65 @@ figure_or_data = {'data': [{'x': [1, 2], 'y': [0, 1]}],
 iplot(figure_or_data)
 ```
 More examples can be found at https://plot.ly/python/animations/.
-- Upcoming animations in online mode: use `plotly.plotly.create_animations` and `plotly.plotly.icreate_animations` which animate a figure with the `frames` argument.
+- animations now work in online mode: use `plotly.plotly.create_animations` and `plotly.plotly.icreate_animations` which animate a figure with the `frames` argument. Here is a simple example:
+```
```

Comment:
🐄 this is a little too much to stick in a changelog (imo) the bullet point is plenty i think. it's more consistent with what we typically do.